go.mod

```diff
@@ -1,32 +1,32 @@
 module github.com/snapcore/snapd
 
-go 1.18
+go 1.23.0
 
 // maze.io/x/crypto/afis imported by github.com/snapcore/secboot/tpm2
 replace maze.io/x/crypto => github.com/snapcore/maze.io-x-crypto v0.0.0-20190131090603-9b94c9afe066
 
 require (
-	github.com/bmatcuk/doublestar/v4 v4.6.1
+	github.com/bmatcuk/doublestar/v4 v4.9.1
 	github.com/canonical/go-efilib v1.6.0
 	github.com/canonical/go-sp800.90a-drbg v0.0.0-20210314144037-6eeb1040d6c3 // indirect
 	github.com/canonical/go-tpm2 v1.13.0
 	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
 	github.com/godbus/dbus/v5 v5.1.0
-	github.com/gorilla/mux v1.8.0
+	github.com/gorilla/mux v1.8.1
 	github.com/gvalkov/golang-evdev v0.0.0-20191114124502-287e62b94bcb
-	github.com/jessevdk/go-flags v1.5.1-0.20210607101731-3927b71304df
-	github.com/juju/ratelimit v1.0.1
-	github.com/mattn/go-runewidth v0.0.15
+	github.com/jessevdk/go-flags v1.6.1
+	github.com/juju/ratelimit v1.0.2
+	github.com/mattn/go-runewidth v0.0.16
```

Comment:
This one is okay: https://salsa.debian.org/go-team/packages/golang-github-mattn-go-runewidth